GitHub-Actions/two-stage-pipeline-template/questions.json

```diff
@@ -19,16 +19,16 @@
     "key": "sam_template",
     "question": "What is the template file path?",
     "default": "template.yaml"
+  }, {
+    "key": "message_stage_name",
+    "question": "We use the stage name to automatically retrieve the bootstrapped resources created when you ran `sam pipeline bootstrap`.\n",
```

Comment:
Probably change this to:
`We use the stage name you used during `sam pipeline bootstrap` to automatically retrieve the bootstrapped resources.\n`